examples/tutorials/data/magic.py

```diff
@@ -1,8 +1,19 @@
 """
-Spectral analysis with energy-dependent directional cuts
-========================================================
+MAGIC with Gammapy
+==================
 
-Perform a point like spectral analysis with energy dependent offset cut.
+Explore the MAGIC IRFs and perform a point like spectral analysis with energy dependent offset cut.
+
+
+Introduction
+------------
+
+`MAGIC <https://magic.mpp.mpg.de/>`__ (Major Atmospheric Gamma Imaging
+Cherenkov Telescopes) consists of two Imaging Atmospheric Cherenkov telescopes in La Palma, Spain.
+These 17m diameter telescopes detect gamma-rays from ~ 30 GeV to 100 TeV.
+This notebook provides an introduction to using the MAGIC DL3 data products, to produce a
+`~gammapy.datasets.SpectrumDataset`. Importantly it shows the uses how to a spectral analysis
+with energy-dependent directional cuts. This is described further below.
```

Comment:
```suggestion
with energy-dependent directional cuts. This is described further below.

For reference, see `Standardised formats and open-source analysis tools for the MAGIC telescopes data <https://ui.adsabs.harvard.edu/abs/2024JHEAp..44..266A/abstract>`__.
```
Here I propose adding the reference to the MAGIC work describing the standarization of their data products and analysis validation using Gammapy.